runner/executer.go

```diff
@@ -0,0 +1,67 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/cybozu-go/github-actions-controller/metrics"
+)
+
+type Executer interface {
+	RunConfigure(ctx context.Context, configArgs []string) error
+	RunService(ctx context.Context) error
```

Comment:
The names of these functions know too much about the internal implementation.
For example, what about the names `Configure` and `Listen`?